### DISCUSSION TITLE
form.enablemultiline() is generating a pdf with too big linehight

### DISCUSSION THREAD

**Discussion by Rajasekar24** [Hopding/pdf-lib#1207] [Q&A]
Hi,

am using this wonderful library for filling a pdf form file using a pure js code. 

first i have a PDF file with ready form fields, and am using the field name to have the value using 
form.setText() then i do enable the multiline using the enableMultiline()

the output is always genertaing multilines as expected BUT its leaving too much space between each line 

code sample :
```
form.getTextField("specify").setText(data);
form.getTextField("specify").enableMultiline();
```

would someone help please ?
![final output generated pdf](https://user-images.githubusercontent.com/43476693/161955728-49de7ce4-4ffb-464f-9ac5-81beb2f72825.png)
![adobe properties](https://user-images.githubusercontent.com/43476693/161955808-e63bff14-16dc-462d-8f98-301553053141.png)
![space that need to be removed](https://user-images.githubusercontent.com/43476693/161955862-32e69db6-d25a-4ac4-8f19-4da12c508eb5.png)

**Comment by dan-willett** [comment]
I did not see it set the line height for the text field anywhere, leaving whatever is the default height with no way to specify the line height for text fields

Another related issue is with layoutMultilineText which is used by the PDF text field in PDF-Lib...  layoutMultilineText does not have the option of specifying the line height, instead the line height is hard coded as font height + height * 0.2.  ( which would be better coded as height * 1.2 )
